### ISSUE TITLE
When you listening music for a long time there are sound drops every second.

### ISSUE THREAD

**Issue by shoker537** [digimezzo/dopamine-windows#701]
And there is a hight CPU using (up to 60% when usually 8%)
App restarting helps.
32-bit, Windows 10 Pro

**Comment by kumarharsh** [comment]
Yeah, I am suffering from this too... It starts sounding like the music is stopping for some microseconds and restarting - many many times. It starts sounding like the music is playing in slow-motion.

**Comment by ghost** [comment]
am human, can confirm this is happening to me as well (64-Bit, Windows 10 Pro. Dopamine 1.5.14)

**Comment by Keith94** [comment]
Maybe system specs and steps to reproduce could help to diagnose the issue.

**Comment by ghost** [comment]
can't say for everyone else but here's my specs: AMD FX-8320 Octa-core @3.5GHz, NVIDIA GeForce GTX 1060 (6GB GDDR5), & 32GB(4x8GB) of DDR3 RAM

For steps, well basically just run non-stop music for somewhere around 30min-2hr

**Comment by Eqnesto** [comment]
Same here. It's really annoying.

**Comment by daedos** [comment]
Using V2 Preview 3 and get this too.  CPU usage ~10% and after 3 albums, I can't do anything on my computer without the music stuttering.  Shame as Dopamine is great player otherwise.

**Comment by daedos** [comment]
> Using V2 Preview 3 and get this too. CPU usage ~10% and after 3 albums, I can't do anything on my computer without the music stuttering. Shame as Dopamine is great player otherwise.

Happy to say v2.0 preview 4 is working great for me.  Well done.

**Comment by digimezzo** [comment]
@daedos, thanks for getting back to me on this. I did put effort in improving performance in preview 4 and 5. I'm glad it's paying off.

**Comment by wetfly** [comment]
started using Dopamine (latest preview) a week or two ago and having the same problem

restarting dopamine isn't a huge bother for me but obviously still an issue

**Comment by mohit301997** [comment]
I am also facing sound quality drop issue.It is not limited to dopamine but other music players gets this issue.Please fix this issue.Meanwhile AIMP is the best music player in every way.

**Comment by eNV25** [comment]
I think this is related to memory use.
I noticed that in the beginning memory use was 99Mb and as I went through more and more songs (press next song repetitively) memory use keeps on increasing, even until 200Mb, but never returns to 99Mb. I think the more song you play the worse it gets.
If this helps, most of my songs are 200kbps-300kbps and most songs have corresponding .lrc files.
